test/test_public_bindings.py

```diff
@@ -99,6 +99,7 @@ def test_no_new_bindings(self):
             "device",
             "DeviceObjType",
             "DictType",
+            "DisableTorchFunction",
```

Comment:
Do we really want this to be in `torch.` ? Not `torch.overrides` or another torch_function related namespace?

Reply:
So at minimum I argue that DisableTorchFunction and DisableTorchFunctionSubclass should be in the same place (either `torch.overrides` or `torch.`). I can put up an issue but given how hard it was to land #89221 which was just a rename I'm not confident that I can move both easily

Reply:
Ok, let's do it this way then...